Makefile

```diff
@@ -54,10 +61,10 @@ example_run_direct_table: requirements.txt ## Run the profile directly, job with
 	poetry run python src/ai/whylabs/templates/$(TEMPLATE).py \
 		--job_name="$(JOB_NAME)" \
 		--input-mode=BIGQUERY_TABLE \
-		--input-bigquery-table=bigquery-public-data:hacker_news.comments \
-		--date-column=time_ts \
+		--input-bigquery-sql=bigquery-public-data.hacker_news.full \
```

Comment:
This looks like a mistake, it should still be table right? -sql would be used for an actual sql query.

Reply:
correct, fixed it now